pl_examples/basic_examples/mnist_examples/image_classifier_4_lightning.py

```diff
@@ -0,0 +1,85 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
rename as `_4_lightning_module`

Reply:
you mean remove "image_classifier"? Why?
@tchaton

Reply:
I would prefer not, this would look weird.

Reply:
`image_classifier_4_lightning_module` just have this with module as it is introducing module

Reply:
Done !